models/speech_commands_v0.01/Cinnamon/test_results.json

```diff
@@ -0,0 +1,122 @@
+[
+  {
+    "accuracy_test": 0.8445999999999999,
+    "accuracy_val": 0.8882665094339622,
+    "architecture": "GRU",
+    "filename": "GRU60KeywordSpotter.pt",
+    "hard_sigmoid": false,
+    "hidden_size": 60,
+    "input_size": 512,
+    "num_filters": 80,
+    "num_layers": 2,
+    "predict_time": 0.236927,
+    "reset": false,
+    "sample_rate": 16000,
+    "vad": false
+  },
+  {
+    "accuracy_test": 0.8734000000000001,
+    "accuracy_val": 0.8882665094339622,
```

Comment:
How come the accuracy_val is the same between the same models with reset = True and reset = False, but the accuracy_test is different. Does that mean the validation never uses reset?

Reply:
ah, because accuracy_val is only measured by pytorch at training time, so it is not something that is "tested" with and without model reset.  It is only the "test" run where I'm testing the model with ELL that we can test the true/false permutations of the triple [reset, vad, hardsigmoid].

Reply:
Makes sense - thanks!